README.md

```diff
@@ -50,6 +50,29 @@ You can try out CBL-Mariner with the following ISO: [x86_64 ISO](https://osrelea
 1. *[Optional]* Shut down VM and adjust boot order to place Hard Drive first. Otherwise if Network boot occurs, your VM will attempt to PXE boot with a timeout of about 2 minutes before trying the next boot option.
 1. When prompted sign in to your CBL-Mariner system using the user name and password provisioned through the Installer.
 
+### Verify ISO via Checksum
+We also provide the ISO checksum in a file and GPG signature of that file so you can independently verify the contents.
+
+Steps:
+1. Install `gpg` onto your system.
+   - On Ubuntu, you can run: `sudo apt-get install gpg`
+1. Download Mariner RPM key from the GitHub repo: [Mariner RPM key](https://github.com/microsoft/CBL-Mariner/blob/1.0/SPECS/mariner-repos/MICROSOFT-RPM-GPG-KEY)

```

Comment:
Is it possible to use the `1.0-stable` tag instead of the `1.0` branch for the URL? It's probably a bit safer this way.

Reply:
Fixed in latest